src/OrchardCore.Modules/OrchardCore.Users/OrchardCore.Users.csproj

```diff
@@ -29,9 +29,9 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Navigation.Core\OrchardCore.Navigation.Core.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Recipes.Abstractions\OrchardCore.Recipes.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement\OrchardCore.ResourceManagement.csproj" />
-    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Roles.Core\OrchardCore.Roles.Core.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Settings.Core\OrchardCore.Settings.Core.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Setup.Abstractions\OrchardCore.Setup.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Workflows.Abstractions\OrchardCore.Workflows.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.Roles\OrchardCore.Roles.csproj" />
```

Comment:
We don't reference modules from modules.

Reply:
@sebastienros 
The problem here is that the `RoleDocument` field belongs to the `OrchardCore.Roles` project and if we move it to `OrchardCore.Roles.Core` we would have to decorated it with `[YesSql.SimplifiedTypeName("OrchardCore.Roles.Models.RolesDocument, OrchardCore.Roles")]` which is not something you'll consider